turtlesim/tutorials/teleop_turtle_key.cpp

```diff
@@ -145,20 +243,30 @@ int main(int argc, char** argv)
 
   int rc = teleop_turtle.keyLoop();
   input.shutdown();
-  rclcpp::shutdown();
```

Comment:
Why is this being removed?

Reply:
It shouldn't have been, my mistake. Restored 55c5aa4